Enforce max col width (fixes #46)

tepi opened the pull request:
<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/vaadin/components/91)
<!-- Reviewable:end -->

tomivirkki commented:
Missing a test & changes, needs a manual rebase

---







Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/components/91)
<!-- Sent from Reviewable.io -->

tepi commented:
Added changes. As for the test, there already was one but a useless one since the content was narrower than the maxWidth set. Fixed it by changing content to be wider.

---







Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/components/91)
<!-- Sent from Reviewable.io -->

tomivirkki commented:
Reviewed 3 of 3 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/components/91)
<!-- Sent from Reviewable.io -->
